tck/pom.xml

```diff
@@ -62,44 +63,90 @@
         <arquillian.junit5.version>1.7.0.Alpha5</arquillian.junit5.version>
     </properties>
 
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>jakarta.json</groupId>
+                <artifactId>jakarta.json-tck-common</artifactId>
+                <version>${project.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>jakarta.json</groupId>
+                <artifactId>jakarta.json-tck-ug</artifactId>
+                <version>${project.version}</version>
+            </dependency>
+
+            <dependency>
+                <groupId>jakarta.json</groupId>
+                <artifactId>jakarta.json-api</artifactId>
+                <version>2.1.0-SNAPSHOT</version>
```

Comment:
`${project.version}` ?

Reply:
not necessarily - tck and api version may be different/have different lifecycle - ie does minor update of the spec jar require and update of TCK?

Reply:
True, it can have a different lifecycle. Never mind how confusing that would be.